Tasks/google-play-release/GooglePlay.js

```diff
@@ -89,10 +89,11 @@ currentEdit = currentEdit.then(function (res) {
 });
 
 if (shouldAttachMetadata) {
+    var metadataRootPath = tl.getInput("metadataRootPath", false);
```

Comment:
is it guaranteed that the path will always be present? May be validate before `addMetadata` that the value was provided?
PS: Could this be a minimatch version that we might need to resolve?